README.md

```diff
@@ -9,223 +9,32 @@ _Just Another System for Epityping NGS data_
 >[!WARNING]
 >**JASEN is in beta stage and the results are unverified. There is no guarantee that the pipeline can execute, output format consistency, or that it produces accurate results until there is an official 1.0 release.**
 
-Jasen produces results for epidemiological and surveillance purposes.
-Jasen has been developed for a small set of microbiota (primarily MRSA), but will likely work with any bacteria with a stable cgMLST scheme.
+Jasen produces results for antibiotic resistance and virulence prediction and epidemiological typing for surveillance purposes. The pipeline is developed in collaboration with several Swedish hospitals. The development was funded by [Genomic Medicine Sweden](https://genomicmedicine.se/).
 
-## Requirements
-
-* [Singularity](https://docs.sylabs.io/guides/3.0/user-guide/installation.html#install-on-windows-or-mac)
-* [JRE 8 - 21](https://www.java.com/en/download/manual.jsp)
-* Nextflow (`curl -s https://get.nextflow.io | bash`)
-
-### Recommended
-
-* Conda
-* Singularity Remote Login
-
-## Usage
-
-### Simple self-test
-
-```
-nextflow run main.nf -profile staphylococcus_aureus -config configs/nextflow.base.config --csv assets/test_data/samplelist.csv
-```
-
-#### Usage arguments
-
-| Argument type | Options                                | Required |
-| ------------- | -------------------------------------- | -------- |
-| -profile      | **staphylococcus_aureus**, escherichia_coli, klebsiella_pneumoniae, mycobacterium_tuberculosis| True     |
-| -config       | **configs/nextflow.base.config**, configs/nextflow.dev.config, configs/nextflow.hopper.config, configs/nextflow.ngp.config| True     |
-| -entry        | bacterial_default                      | True     |
-| --output      | User specified directory                         | False    |
-| -resume       | Not applicable                                     | False    |
-
-
-### Input file format 
-
-```csv
-id,platform,read1,read2
-p1,illumina,assets/test_data/sequencing_data/saureus_10k/saureus_large_R1_001.fastq.gz,assets/test_data/sequencing_data/saureus_10k/saureus_large_R2_001.fastq.gz
-```
-
-### Update databases
-
-#### Update MLST database
-
-```
-bash /path/to/jasen/assets/mlst_db/update_mlst_db.sh
-```
+The pipeline currently support a small set of microbiota and the support are in different stages of development. See the documentation of information on the supported analysis for each species and what the development status means.
 
+| Species                      | Development status |
+|------------------------------|--------------------|
+| *Staphylococcus arueus*      | Draft              |
```

Comment:
Do you reckon this is true? Should we not say beta?

Reply:
The exact nomenclature doesnt matter to me. But I think draft is perhaps more expressive than beta. We also define the difference between the different statuses in the text as well.